bq_dts/helpers.py

```diff
@@ -179,6 +181,9 @@ def normalize_transfer_run(transfer_run, integer_params=None):
 
 def protobuf_struct_to_python_dict(raw_struct):
     # https://developers.google.com/protocol-buffers/docs/reference/google.protobuf#struct
+    if raw_struct is None:
+        return raw_struct
```

Comment:
return None ?

Reply:
This is the case where raw_struct is passed in as none. Could return an empty dict as well, but the rest of the code did check.